README.md

```diff
@@ -17,16 +14,12 @@ npm install jazzicon -S
 
 Takes a pixel diameter and a javascript integer (seeds the shape), and gives you back a DOM element to use as you wish!
 
-```javascript
-var jazzicon = require('jazzicon')
+```typescript
+import jazzicon from 'jazzicon'
```

Comment:
Is it `@metamask/jazzicon`?

Reply:
feel free to branch off this and change anything, i sent the pr a while ago